app/models/crime_application.rb

```diff
@@ -34,5 +36,5 @@ class CrimeApplication < ApplicationRecord
 
   # Before submitting the application we run a final
   # validation to ensure some key details are fulfilled
-  validates_with ApplicationFulfilmentValidator, on: :submission
+  validates_with ApplicationFulfilmentValidator, on: :submission, unless: :post_submission_evidence?
```

Comment:
Do we need a new validator? I guess as these are straightforward then perhaps not.